Source/common/SNTConfigurator.m

```diff
@@ -459,6 +459,23 @@ + (NSSet *)keyPathsForValuesAffectingEnableBadSignatureProtection {
   return [self configStateSet];
 }
 
++ (NSSet *)keyPathsForValuesAffectingBlockUSBMount {
+  return [self syncAndConfigStateSet];
+}
+
++ (NSSet *)keyPathsForValuesAffectingBannedUSBBlockMessage {
+  return [self syncAndConfigStateSet];
+}
+
++ (NSSet *)keyPathsForValuesAffectingRemountUSBBlockMode {
+  return [self syncAndConfigStateSet];
+}
+
++ (NSSet *)keyPathsForValuesAffectingRemountUSBBlockMessage {
+  return [self syncAndConfigStateSet];
+}
+
+
```

Comment:
I also see a property `usbBlockMessage`, we should probably have a `keyPathsForValuesAffectingUsbBlockMessage` too

Reply:
It needs to be "AffectingUsbBlockMessage" - the property is `usbBlockMessage`